src/finch/tensor.py

```diff
@@ -295,6 +295,13 @@ def _is_dense(self) -> bool:
     def _order(self) -> tuple[int, ...]:
         return jl.typeof(self._obj).parameters[1]
 
+    @property
+    def mT(self) -> "Tensor":
+        axes = list(range(self.ndim))
+        axes[-2:] = axes[:-3:-1]
```

Comment:
```suggestion
        axes[-2], axes[-1] = axes[-1], axes[-2]
```

Reply:
(This is more code but clearer to parse for the uninitiated)

Reply:
Done!